scipy/stats/_continuous_distns.py

```diff
@@ -3792,6 +3797,9 @@ def _rvs(self, b):
         return np.sqrt((t*t).sum(axis=0))
 
     def _pdf(self, x, b):
+        # For numerical stability, use (x**2 + b**2)/2 = ((x-b)**2)/2 + xb.
+        # The factor of exp(-xb) is then included in the i0e function
+        # in place of the modified Bessel function, i0.
```

Comment:
Strictly speaking, it's the other way around: we use `i0e` for stability, and compensate for the extra exponential by rearranging the quadratic form. But this is really minor.